src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -161,6 +159,13 @@ def get_sdk(self):
             self._sdk = [{"sdk": normalized_sdk_tag_from_event(self.event)}]
         return self._sdk
 
+    def get_family(self):
+        # codecov hates implementation with an if statement
```

Comment:
If it makes sense, you can share feedback in #discuss-codecov.

Do you think we need the comment for posterity?

Reply:
Yeah, there is a valid explanation for it. CodeCov requires both branches of `if` to be covered, event if there is no specific `else` branch, because of possible side-effects the `if` can have on the rest of the code. I've updated the comment.